addons/crm/views/crm_lead_views.xml

```diff
@@ -1044,6 +1052,73 @@ if record:
             <field name="act_window_id" ref="crm_lead_action_pipeline"/>
         </record>
 
+        <record id="crm_lead_action_duplicates" model="ir.actions.act_window">
+            <field name="name">Similar Leads</field>
+            <field name="res_model">crm.lead</field>
+            <field name="view_mode">tree,form,kanban</field>
+            <field name="search_view_id" ref="crm.view_crm_case_leads_filter"/>
+        </record>
+
+        <record id="view_crm_lead_duplicates_kanban" model="ir.ui.view">
```

Comment:
Id and name should follow guidelines: crm_lead_view_kanban_duplicates / crm.lead.view.kanban.duplicates for example.